.travis.yml

```diff
@@ -1,27 +1,29 @@
 language: ruby
 gemfile:
-  - gemfiles/rails3_2.gemfile
-  - gemfiles/rails4_0.gemfile
   - gemfiles/rails4_2.gemfile
+  - gemfiles/rails5_0.gemfile
+  - gemfiles/rails5_1.gemfile
+  - gemfiles/rails5_2.gemfile
+  - Gemfile
 rvm:
-  - 2.0.0
-  - 2.1.1
-  - 2.2.4
-  - 2.3.4
-  - 2.4.1
+  - 2.5.7
+  - 2.6.5
+  - 2.7.0
+
+env:
+  matrix:
+    - MONGODB=3.4
+    - MONGODB=3.6
+    - MONGODB=4.0
+
+before_script:
+  - docker run -d -p 27017:27017 mongo:${MONGODB}
+  - docker ps -a
```

Comment:
Is this necessary, or just debug purposes?

Reply:
just for debug purpose, listing the process is not necessary :pray: